src/RecurringDonations/RecurringDonationsCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace GiveTestData\RecurringDonations;
+
+use WP_CLI;
+use GiveTestData\TestData\Framework\ProviderForwarder;
+use GiveTestData\TestData\Factories\DonationFactory as DonationsFactory;
+
+/**
+ * A WP-CLI command for seeding test data.
+ */
+class RecurringDonationsCommand {
+	/**
+	 * @var RecurringDonationsCommand
+	 */
+	private $donationsFactory;
+
+	use ProviderForwarder;
```

Comment:
Traits should be listed before any class properties.

Reply:
Good catch!